### DISCUSSION TITLE
Create snapshot failing on native function in token objects

### DISCUSSION THREAD

**Discussion by gregnazario** [aptos-labs/aptos-developer-discussions#66] [Questions] [answered]
From hari krishna on discord:

> Hi,  I am using  aptos_token_object.create_named_token, 
> I had tests written, it was working 2 weeks ago. Yesterday when I tried the tests, it was failing with the following error. I believe there was an update on the concurrent minting on the framework.  Could this be a result of that ? how can I fix this? Thank you

> Using Aptos 3.0.2 version

```shell
.move/https___github_com_aptos-labs_aptos-core_git_mainnet/aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move:103:23
│     │
│ 103 │     public native fun create_snapshot<Element: copy + drop>(value: Element): AggregatorSnapshot<Element>;
│     │                       ^^^^^^^^^^^^^^^
│     │                       │
│     │                       Test was not expected to error, but it aborted with code 196613 originating in the module 0000000000000000000000000000000000000000000000000000000000000001::aggregator_v2 rooted here
│     │                       In this function in 0x1::aggregator_v2
│


```

**Comment by harispericorn** [comment, marked as answer]
for anyone facing this issue its because of aptos version you need to update dependency rev version to your aptos-cli-version
